### PR TITLE
Changed decimal places to 2 for Public and Private category graphs

### PR DESCRIPTION
**Pull request by ddomdel**
Decimal places are now set to two for Public and Private views
# Before
image

# After
image

I could have opted for `Math.round()` but it would round it off to a whole number. So I went for `toFixed()` but it turns the result to a string. Therefore I enclosed the result in the Number() wrapper object.
image

### CODE REVIEW

**Comment by ddomdel** [comment]
The changes made weren't compiled by Vite. Therefore, to see the changes, it must be compiled by Vite first. @ColdHeat Let me know if we should always be compiling when making updates to Javascript or CSS files. Which will replace old compiled JS and CSS files.

File: assets/js/users/private.js [hunk 1]
```diff
@@ -42,10 +42,11 @@ Alpine.data("UserGraphs", () => ({
 
     let data = [];
     for (const property in breakdown) {
+      let percent_result = (breakdown[property] / categories.length) * 100;
       data.push({
         "name": property,
         "count": breakdown[property],
-        "percent": (breakdown[property] / categories.length) * 100,
+        "percent": Number(percent_result.toFixed(2)),
```
Comment (ColdHeat) [inline assets/js/users/private.js:49]: I don't think you need to call toFixed here actually? Becuase percent_result should already be a number.

```
> 98.967.toFixed(2)
'98.97'
```

File: assets/js/users/public.js [hunk 2]
```diff
@@ -42,10 +42,11 @@ Alpine.data("UserGraphs", () => ({
 
     let data = [];
     for (const property in breakdown) {
+      let percent_result = (breakdown[property] / categories.length) * 100;
       data.push({
         "name": property,
         "count": breakdown[property],
-        "percent": (breakdown[property] / categories.length) * 100,
+        "percent": Number(percent_result.toFixed(2)),
```
Comment (ColdHeat) [inline assets/js/users/public.js:49]: Same as above

**Comment by ColdHeat** [comment]
Yes for now we will be committing those built files for now because we need the repo to work with just `git pull` or a regular download.

**Comment by ColdHeat** [comment]
Closes #12

**Pull request merged**